controllers/awsmachine_controller.go

```diff
@@ -415,13 +415,13 @@ func (r *AWSMachineReconciler) AWSClusterToAWSMachines(o handler.MapObject) []ct
 	}
 
 	labels := map[string]string{clusterv1.MachineClusterLabelName: c.Name}
```

Comment:
Doesn't this assume that the AWSCluster is going to have the same name as the Cluster?

Reply:
This is used in the label matcher below for getting the machines for the cluster in the query below.

Reply:
nm, @vincepri is correct, we are using the name of the AWSCluster here as the Cluster name.